.github/workflows/dance.yml

```diff
@@ -85,11 +86,20 @@ jobs:
       - name: Run init
         run: bin/task init
 
+      # The step that creates or updates the environment variable does not have access to the new value, 
+      # but all subsequent steps in a job will have access.
+      - name: Set GITHUB_ENV
+        run: |
+          if [[ ${{ matrix.test == 'dbaas_core' }} ]]; then
+            echo "FILE_CONCURRENCY=1" >> $GITHUB_ENV
```

Comment:
We do this because processing more than one file concurrently seems to make testing unstable for FerretDB.

I will create a separate issue later for this.

Reply:
Does this only happen on CI? I'm seeing this env var is set on CI but not for local runs.

Reply:
FYI FILE_CONCURRENCY is set to 1 for all jobs due to https://github.com/actions/runner/issues/1126. It cannot be unset so all jobs will be slower overall that depend on the runner.